Mathlib/Probability/Distributions/Gaussian/Real.lean

```diff
@@ -417,15 +408,15 @@ theorem complexMGF_id_gaussianReal (z : ℂ) :
       rw [integral_cexp_quadratic (by simpa using pos_iff_ne_zero.mpr hv), ← mul_assoc]
     _ = 1 * cexp (-μ ^ 2 / (2 * v) - (z + μ / v) ^ 2 / (4 * -(2 * v)⁻¹)) := by
       congr 1
-      field_simp [Real.sqrt_eq_rpow]
+      simp [field, Real.sqrt_eq_rpow, -mul_inv_rev]
```

Comment:
squeeze

Reply:
Naively squeezing this doesn't work. @hrmacbeth Do you feel like investigating? You did this before...